vm/actor/tests/common/mod.rs

```diff
@@ -80,6 +81,18 @@ impl<'a, BS: BlockStore> MockRuntime<'a, BS> {
             actor_code_cids: HashMap::new(),
             new_actor_addr: None,
 
+            message: UnsignedMessage::builder()
+                .to(Address::new_id(100).unwrap())
+                .from(Address::new_id(100).unwrap())
+                .sequence(0) // optional
+                .value(TokenAmount::from(0u8)) // optional
+                .method_num(MethodNum::default()) // optional
+                .params(Serialized::default()) // optional
+                .gas_limit(0) // optional
+                .gas_price(TokenAmount::from(0u8)) // optional
+                .build()
+                .unwrap(),
```

Comment:
Where are you getting those addresses? seems like a bad default. (You only need to specify to, from, and value and value will default to 0 if not used in builder)
```suggestion
                .to(Address::new_id(100).unwrap())
                .from(Address::new_id(100).unwrap())
                .build()
                .unwrap(),
```